docs/02.API-REFERENCE.md

```diff
@@ -2378,6 +2398,46 @@ main (void)
 - [jerry_container_type_t](#jerry_container_type_t)
 
 
+*New in version [[NEXT_RELEASE]]*.
+
+## jerry_get_builtin_object
+
+**Summary**
+
+Get the builtin object.
```

Comment:
Better description: `Get the specified built-in object. The function always returns with an object, unless the feature required for the built-in object is disabled.`